packages/@react-spectrum/illustrated-message/stories/IllustratedMessage.stories.tsx

```diff
@@ -33,7 +35,17 @@ storiesOf('IllustratedMessage', module)
   );
 
 function render(props: any = {}) {
+  let {
+    illustration,
+    heading,
+    description,
+    ...otherProps
+  } = props;
   return (
-    <IllustratedMessage {...props} />
+    <IllustratedMessage {...otherProps}>
+      {illustration}
+      {heading && <Header><Heading>{heading}</Heading></Header>}
```

Comment:
I'm undecided on this way of adding optional header and description, but since this is storybook I'm not sure it matters.

Reply:
FYI, header as a wrapper around heading is unnecessary and a pattern we're getting rid of https://github.com/adobe-private/react-spectrum-v3/pull/312/files#diff-42191d0dcb0b765b7830298bef6def6eR237